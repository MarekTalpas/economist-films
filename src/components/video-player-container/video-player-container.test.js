// @flow
import React from 'react';
import renderer from 'react-test-renderer';
import { shallow } from 'enzyme';
import VideoPlayerContainer from './video-player-container';

describe('VideoPlayerContainer', () => {
  test('renders correctly', () => {
    const tree : string = renderer.create(<VideoPlayerContainer params={{id: 50}} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
