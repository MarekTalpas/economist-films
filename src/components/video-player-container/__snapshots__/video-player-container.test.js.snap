exports[`VideoPlayerContainer renders correctly 1`] = `
<div
  className="video-player-container" />
`;
